test/cpp/jit/test_gpu.cpp

```diff
@@ -2999,7 +2999,7 @@ TEST(NVFuserTest, FusionRFactorReplay_CUDA) {
 
 // Start off simple, block on the outer dim
 // block stride + thread all reduce + unrolling on inner dim
-TEST(NVFuserTest, FusionReduction_CUDA) {
+TEST(NVFuserTest, FusionReduction1) {
```

Comment:
I think we need the `_CUDA` suffix. @jjsjann123 , is that right?

Reply:
Oh, I didn't know that. I thought it's redundant as we now have the `NVFuserTest` prefix.

Reply:
No, pretty confident it's needed.

Reply:
OK, added back the suffix.